### ISSUE TITLE
Adding Leetcode, Codechef solutions to the repository

### ISSUE THREAD

**Issue by Ninjavin** [OpenGenus/cosmos#4343]
This is in accordance with the Cosmos Task List 3. I would like to add the coding solutions to CodeChef, Leetcode Questions to the repo.
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [x] Proposal to the Repository

**Comment by Aakansha99** [comment]
@asha15
I would also like to contribute to the issue as a part of GSSoC'20, could you please assign it to me

**Comment by nitin1kumar1** [comment]
I have solved plenty of questions on Codechef and codeforces. I would like to provide my solutions. I can do both Leetcode and CodeChef. as part of GSSoC , I would like you to assign the task to me.

**Comment by asha15** [comment]
The project mainly focused about the Algorithm and Data Structures. Adding solutions for CodeChef problem is out of scope
@jainaman224 What's your opinion on this?

**Comment by nitin1kumar1** [comment]
> The project mainly focused about the Algorithm and Data Structures. Adding solutions for CodeChef problem is out of scope
> @jainaman224 What's your opinion on this?

Yes, I get that solutions are not needed. But I have used plenty of those algorithms during contest, so I can provide some which are not already present.

**Comment by asha15** [comment]
@nitin1kumar1 so choose one algorithm which was not presented in the repo and create a issue for it, Will happy to assign it for you

**Comment by shristi428** [comment]
i like to solve leetcode questions and try to solve as many as question everyday  and i already have a repo for my leetcode solutions please assign me this problem

**Comment by anishajain22** [comment]
Can I  please take up the Leetcode issue. I have solved many problems on Leetcode and can provide the solutions for the same.

**Comment by Salonii02** [comment]
I would also like to contribute to the issue as a part of GSSoC'20, could you please assign it to me
@Ninjavin

**Comment by arnavb** [comment]
To all the people who commented here, contributing these things is fine, just that it should be one issue per problem, not all problems associated with this issue.
